fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroup.java

```diff
@@ -71,14 +71,18 @@ public class WorkloadGroup implements Writable, GsonPostProcessable {
 
     public static final String SPILL_THRESHOLD_HIGH_WATERMARK = "spill_threshold_high_watermark";
 
+    public static final String TAG = "tag";
+    public static final String TAG_DEFAULT_VALUE = "default_tag";
```

Comment:
Do not use this logic to manage workload group without tag. 
If a workload's tag is empty, then it is on all be.